app/src/main/res/layout/profile_deletion_dialog.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ditto for this file--let's remove if we aren't using it anywhere in this PR.

Reply:
Removed.